File: market-mapper/src/main/java/by/market/mapper/ContainerMetadataMapper.java
package by.market.mapper;

import by.market.domain.system.ContainerMetadata;
import by.market.dto.system.ContainerMetadataDTO;
import org.mapstruct.Mapper;

@Mapper(config = MapperConfiguration.class)
public interface ContainerMetadataMapper extends MapstructMapper<ContainerMetadataDTO, ContainerMetadata> {
}

File: market-mapper/src/main/kotlin/by/market/mapper/system/ContainerMetadataMapper.kt
package by.market.mapper.system

import by.market.domain.system.ContainerMetadata
import by.market.dto.system.ContainerMetadataDTO
import by.market.mapper.IMapstructMapper
import by.market.mapper.MapperConfig

import org.mapstruct.Mapper

@Mapper(config = MapperConfig::class)
interface ContainerMetadataMapper : IMapstructMapper<ContainerMetadataDTO, ContainerMetadata>
